src/pages/instances/Events.tsx

```diff
@@ -26,6 +28,8 @@ const Events: FC = () => {
       eventCallback.onFinish?.();
       eventQueue.remove(event.metadata.id);
     }
+
+    setTimeout(() => void refetchOperations(), 3000);
```

Comment:
Why adding the refetch here with a 3s delay?

Reply:
This is related to the previous comment. When an event is handled, I found that if we refetch operations immediately then we might get back the same results from the server, it almost feels like operations on the backend isn't cleared immediately when events are emitted. The combination of delays on the event handling as well as refetching operations allows for both to keep in sync. The 3s delay is an empirical result from experimenting really.